const test = function() {
  throw new Error('test')
}

test()
